client/client/app/app.controller.js

```diff
@@ -234,9 +283,13 @@ export default class ngbAppController extends baseController {
     initStateFromParams() {
         this._changeStateFromParams(this.$stateParams);
 
-        const {toolbar, layout, bookmark, screenshot, embedded} = this.$stateParams;
+        const {toolbar, layout, bookmark, screenshot, embedded, ...rest} = this.$stateParams;
         if (embedded) {
             this.appearanceContext.embedded = this.dictionaryState.on.toLowerCase() === embedded.toLowerCase();
+            const embeddedOptions = Object.entries(rest).filter(([key, value])=> this.appearanceContext[key] !== undefined);
+            embeddedOptions.forEach(([key, value]) => {
+                this.appearanceContext[key] = value && this.dictionaryState.on.toLowerCase() === value.toLowerCase();
+            });
```

Comment:
(notice comments below - `embedded` means "hide all actions")
We need to set each option individually if `embedded` (or `hideAll`) is **not** set